iotileemulate/test/test_reference.py

```diff
@@ -20,6 +22,25 @@ def reference():
     device.stop()
 
 
+@pytest.fixture(scope="function")
+def reference_hw():
+    """Get a reference device and connected HardwareManager."""
+
+    device = ReferenceDevice({})
+    peripheral = EmulatedPeripheralTile(11, b'abcdef', device)
+    peripheral.declare_config_variable("test 1", 0x8000, 'uint16_t')
+    peripheral.declare_config_variable('test 2', 0x8001, 'uint32_t[5]')
+
+    device.add_tile(11, peripheral)
+
+    adapter = EmulatedDeviceAdapter(None, devices=[device])
```

Comment:
@mattrunchey For tests that you're writing in CoreTools, here's a way to avoid needing to serialize states into base64 and pass a giant string or create json files.  You can just directly create the virtual device classes and pass them into a precreated `VirtualDeviceAdapter` instance.